recipes/libq/all/conanfile.py

```diff
@@ -0,0 +1,108 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import collect_libs, copy, get, rm, rmdir, replace_in_file
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.53.0"
+
+class libqConan(ConanFile):
+    name = "libq"
+    description = "A platform-independent promise library for C++, implementing asynchronous continuations."
+    license = "Apache-2.0"
+    topics = ("async", "promises")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/grantila/q"
+    settings = "os", "compiler", "build_type", "arch"
+    package_type = "library"
+    options = {
+        "shared": [True, False]
+    }
+    default_options = {
+        "shared": False
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "apple-clang": "10",
+            "clang": "7",
+            "gcc": "7"
+        }
+
+    @property
+    def _build_tests(self):
+        return not self.conf.get("tools.build:skip_test", default=True, check_type=bool)
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self._build_tests:
+            self.test_requires("gtest/1.14.0")
```

Comment:
```suggestion
```

I would request removing testing requirements if possible. I see it required extra steps to make it work and even in the template there is no unit tests listed.